src/windows/research.c

```diff
@@ -388,7 +388,7 @@ void window_research_development_page_paint(rct_window *w, rct_drawpixelinfo *dp
 	int lastDevelopmentFormat;
 	if (typeId != 0xFFFFFFFF) {
 		if (typeId >= 0x10000) {
-			rct_ride_type *rideEntry = RCT2_GLOBAL(0x009ACFA4 + (typeId & 0xFF) * 4, rct_ride_type*);
+			rct_ride_type *rideEntry = RCT2_GLOBAL(RCT2_ADDRESS_RIDE_ENTRIES + (typeId & 0xFF) * 4, rct_ride_type*);
```

Comment:
Thanks! Was just about to ask for these two. Only `RCT2_ADDRESS_RIDE_ENTRIES` remaining are in `addresses`, `object_list` and `ride`.